problem1.py

```diff
@@ -1 +1,26 @@
+"""
+-------------------------------------------------------------------------------
+Name:   main.py
+Purpose:  count wins or smth idk
 
+Author: me
+
+Created:  date in today
+------------------------------------------------------------------------------
+"""
+###the range  and counter###
+print("tournament log")
+for x in range(1,7):
+  result = str(input("W or L: "))
+  if result == "W": 
+     win = win + 1
+###supposed to track what bracket you are in###
+if result == 5 or result == 6:
```

Comment:
`win` is the variable you are using to track win counts, not `result`.  The logic here is incorrect